server/service/orbit.go

```diff
@@ -423,13 +424,20 @@ func (svc *Service) SetOrUpdateDeviceAuthToken(ctx context.Context, deviceAuthTo
 	// this is not a user-authenticated endpoint
 	svc.authz.SkipAuthorization(ctx)
 
+	if len(deviceAuthToken) == 0 {
```

Comment:
Should we also check the same in `ds.SetOrUpdateDeviceAuthToken`? (If empty string, then return error)

Reply:
Seems unnecessary to have this double validation. This is the only endpoint using that DB method.